src/codeu/chat/util/Time.java

```diff
@@ -66,4 +66,10 @@ public String toString() {
 
   public static Time now() { return Time.fromMs(System.currentTimeMillis()); }
 
+  // Difference of end and start times.
+  // Ratio is the power of ten from seconds that the duration will be given in
+  // (e.i. ratio = -3 is milliseconds)
+  public static long duration(Time start, Time end, int ratio) {
+    return Math.round(Math.pow(10, -(3 + ratio)) * ((end.inMs() - start.inMs())));
```

Comment:
I'll change it to make it more readable. However, I feel it could be a useful to have it be general. Right now, this is the only code that given two times it converts to a duration, and I feel like it could be useful to allow for a duration to be of any type. Maybe for some applications we will need to get a duration with different ratios (e.g lag between users in microseconds) or something similar. This could probably be its own immutable class though. That would help keep track of the units for arithmetic between two durations for example. What do you think?